<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="LoginPage">
       <layout class="QVBoxLayout" name="loginLayout">
        <item>
         <widget class="QLineEdit" name="username_input">
          <property name="placeholderText">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="password_input">
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="placeholderText">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="login_button">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="CurrencyConverter">
       <layout class="QVBoxLayout" name="converterLayout">
        <item>
         <widget class="QLineEdit" name="amount_input">
          <property name="placeholderText">
           <string>Enter amount</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="from_currency_label">
          <property name="text">
           <string>From Currency:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="from_currency"/>
        </item>
        <item>
         <widget class="QLabel" name="to_currency_label">
          <property name="text">
           <string>To Currency:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="to_currency"/>
        </item>
        <item>
         <widget class="QLabel" name="result_label">
          <property name="text">
           <string>Result: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="convert_button">
          <property name="text">
           <string>Convert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clear_button">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="logout_button">
          <property name="text">
           <string>Logout</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
